### ISSUE TITLE
Sobre o escopo do plugin

### ISSUE THREAD

**Issue by luizbills** [claudiosanches/woocommerce-extra-checkout-fields-for-brazil#152]
Agora com esse nome mais "amplo", você vai seguir com aquela ideia de colocar o plugin mais funcionalidades ligadas a ecommerce brasileiros como simulador de frete?

**Comment by mariovalney** [comment]
Também tenho interesse em ajudar no desenvolvimento o/

Um plugin focado no e-commerce brasileiro é bem interessante.

**Comment by claudiosanches** [comment]
@luizbills sim, essa é a ideia, também adicionar validação e prevenir mais de uma conta com o mesmo CPF e por ai vai.

**Comment by luizbills** [comment]
@claudiosanches devíamos então já pensar em mudar o nome do menu de configurações ou inserir as opções dentro das configurações do woocommerce nas seções mais adequadas (porém isso pode gerar confusão aos usuários).

**Comment by denisoncarlos** [comment]
Creio que uma nova funcionalidade interessante, pensando no E-Commerce Brasileiro, seria acrescentar mais uma Pessoa Jurídica, ou, incluir uma forma de categorizá-la como "Cliente Final" ou como "Revenda/Distribuidora". Observei essa necessidade com empresas distribuidoras, que vendem para revendedores, outros distribuidores menores e consumidores finais (que podem ser pessoas físicas ou jurídicas). Em um caso como esse, há 3 tabelas de impostos devido ao burocrático sistema de substituição tributária de ICMS entre os Estados.

Hoje, para que o cálculo desses impostos sejam feitos de forma correta, é preciso usar um plugin Premium (Wholesale). Nele cria-se esses 3 tipos de clientes nele e os vincula às tabelas de impostos do Woo criadas para cada um. Há ainda a necessidade de manter o plugin do Claudio porque é o único que valida CPF e CNPJ, não havendo nenhuma solução para isso por meio de Campo Personalizado, por exemplo.

Seria muito interessante também, se nesse cenário já fosse possível relacionar cada tipo de usuário à sua tabela de impostos. O gestor da loja criaria as tabelas de impostos para cada tipo de cliente, incluindo as alíquotas como determina a Contabilidade da empresa. Depois, vincularia cada uma ao seu respectivo cliente. Isso eliminaria "um monte de plugins" e manteria o WP rodando mais "enxuto".

Claro que posso ter dito bobagem ao pensar em tanta funcionalidade assim num plugin gratuito. Talvez isso crie uma versão Premium dele. Contudo, seria uma versão Premium que já nasceria com credibilidade, haja visto que os plugins do Claudio são reconhecidos pela qualidade e simplicidade de configuração que possuem.

Se precisarem de informações sobre isso, posso passar para vocês! ;)

**Comment by claudiosanches** [comment]
@denisoncarlos só pra deixar claro, não tem bobagem, entretanto eu nunca vou fazer um plugin premium, pago ou algo assim, se os usuários querem muitas features, é melhor que eles me ajudem desenvolver e manter o plugin, eu não quero dinheiro, eu quero gente me ajudando a manter ele atualizando, quando mais gente me ajudando escrever código melhor vai ser.

**Comment by ogafanhoto** [comment]
> @luizbills sim, essa é a ideia, também adicionar validação e prevenir mais de uma conta com o mesmo CPF e por ai vai.

Olá pessoal!

Tenho procurado por este recurso "prevenir mais de uma conta com o mesmo CPF", mas não consigo encontrar uma solução...

Alguém poderia me ajudar com esta questão?

Agradeço desde já!
